nikola/plugin_categories.py

```diff
@@ -493,6 +493,17 @@ class Taxonomy(BasePlugin):
         in the metadata. If set to None, the classification won't be stored
         in the metadata.
 
+    metadata_friendly_name = None:
+        The name to be used when storing the friendly names of the classification
+        in the metadata. If set to None, the friendly names won't be stored
```

Comment:
You could make a new Post attribute to store those friendly names, or classification-related stuff. Some things might depend on post.meta not changing, and `post.meta` should contain only the raw(-ish) information in post meta tags and nothing else.